RELEASING.md

```diff
@@ -61,7 +61,9 @@ convention of `v<major>.<minor>.x`, while the tags include the patch version
 `v<major>.<minor>.<patch>`. For example, the same branch `v1.7.x`
 would be used to create all `v1.7` tags (e.g. `v1.7.0`, `v1.7.1`).
 
-1. For `master`, change root build files to the next minor snapshot (e.g.
+1. Check current release's [milestone](https://github.com/grpc/grpc-java/milestones)
```

Comment:
I think this isn't in here because nobody is "permitted" to delay the branch cut. We might choose to be nice to people, but at the end of the day there _is never_ a branch-cut blocker.

How about we rephrase this to be "aware" of important issues? Basically, if the first time you've looked for blockers is during "Tagging the Release," then things aren't probably going to go well. So maybe we can suggest to start looking at it here, and hey, if you choose to pause the branch cut, that's up to you.